sql/views.sql

```diff
@@ -37,13 +55,14 @@ compht_size as
  where map.chunk_id = srcch.id and srcht.id = srcch.hypertable_id
  group by srcht.id
 )
-select hts.table_schema, hts.table_name, hts.table_owner, 
+select hts.table_schema, hts.table_name, hts.table_owner,
```

Comment:
Sorry, just noticed:
```suggestion
SELECT hts.table_schema, hts.table_name, hts.table_owner,
```